src/screens/Login/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import './style.css';
+import '../../style.css';
+import wbooks from '../../assets/wbooks_logo.svg';
+import FormErrors from './components/FormErrors';
+import { Redirect } from 'react-router-dom'; 
+import axios from '../../config/api'
+
+class Login extends Component {
+  state = {
+    email: '',
+    pass: '',
+    formErrors: {email: '', pass: ''},
+    emailValid: false,
+    passValid: false,
+    formValid: false,
+    login: false
+  }  
+  
+  handleUserInput = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({[name]: value},
+                   () => { this.validateField(name, value) 
```

Comment:
Why do you need to validate after the state changed?

You should be able to use name and value instead of state.

Reply:
Same with validateForm.

Actually, `formValid` should not be in state since it can be calculated (`emailValid  || passValid`)